### DISCUSSION TITLE
Help me please

### DISCUSSION THREAD

**Discussion by EmpoweredPhoenix** [3kh0/ext-remover#552] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
is there a way to enable crosh with bookmarklets if so please tell me

**Comment by Aka-but-nice** [comment, marked as answer]
Bookmarklets has nothing to do with crosh, tell me your blocker/extension.
I would give you steps for, Point Blank.

**Reply by Overreationary** [reply]
Mine's GoGuardian

**Reply by cooltoo37** [reply]
use point black
